lib/src/text_widget_extensions.dart

```diff
@@ -0,0 +1,98 @@
+import 'package:flutter/gestures.dart';
+import 'package:flutter/material.dart';
+
+extension TextWidgetExtensions on Text {
+  ///Extensions method to make specific part of [Text] clickable or in different style.
+  ///
+  ///Example:
+  ///         Text(
+  ///           "Accept term & conditions to continue",
+  ///         ).withSpacialParts([
+  ///           Part(
+  ///             text: "Term & conditions",
+  ///             onClick: () {
+  ///               print("term & condition clicked");
+  ///             },
+  ///             style: const TextStyle(color: Colors.blue),
+  ///           ),
+  ///         ])
+  Widget withSpacialParts(List<Part> parts) {
```

Comment:
I think it should be changed to `withClickableParts`

Reply:
I named it as a special part because it is also used for style changes; however, if you still feel that it should be clickable, I will make the necessary changes.